tests/swagger.yaml

```diff
@@ -6,6 +6,9 @@ info:
     [Learn about Swagger](http://swagger.wordnik.com) or join the IRC channel `#swagger` on irc.freenode.net.
 
     For this sample, you can use the api key `special-key` to test the authorization filters
+    
+    NOTE: Swagger files mey contain non-acsii characters like µ, ü or ç and ñ, please save your file in UT8 to preserv them.
```

Comment:
Replace with `NOTE: Swagger files may contain non-ascii characters like µ, ü or ç and ñ, please save your file in UTF-8 to preserve them.` please